topology/Makefile.am

```diff
@@ -9,9 +9,23 @@ endif
 	rst2man $< > $@
 
 alsatplg_SOURCES = topology.c pre-processor.c pre-process-class.c pre-process-object.c \
-		    pre-process-dapm.c pre-process-dai.c
-
-noinst_HEADERS = topology.h pre-processor.h
+		pre-process-dapm.c pre-process-dai.c intel/nhlt.c intel/dmic/dmic-nhlt.c \
```

Comment:
Can commit message also contain a link to the NHLT spec. 
https://01.org/sites/default/files/595976_intel_sst_nhlt.pdf
We also need to state the use case, e.g. NHLT data will be build able via topology for firmware consumption.